java/src/jmri/jmris/simpleserver/SimpleServerFrame.java

```diff
@@ -7,7 +7,7 @@
 import javax.swing.JToggleButton;
 
 /**
- * Frame displaying start/stop buttons for the JMRI server.
+ * Frame displaying start/stop buttons for the JMRI simple server.
```

Comment:
would `Simple Server` catch the eye better?  (Not a native speaker, often not sure about capitalization in cases like this)

Reply:
Probably.